app/assets/stylesheets/views/_notes.scss

```diff
@@ -0,0 +1,5 @@
+#noteable_type label,
```

Comment:
Selector `noteable_type` should be written in lowercase with hyphens<br>Selector `noteable_selector` should be written in lowercase with hyphens